test/cli.js

```diff
@@ -86,6 +89,16 @@ yargs
           description: 'Stop on first error.',
           type: 'boolean',
         })
+        .option('debug', {
+          alias: 'd',
+          description:
+            'Allows attaching a debugger and waits for the debugger to start code execution.',
+          type: 'boolean',
+        })
+        .option('inspecting', {
```

Comment:
would `caught-exception` and alias `c` be better?

Reply:
`caught-exception` is something you configure in the inspector. It would also allow breaking on uncaught, breaking on `debugger` and breaking on custom breakpoints. 

So `caught-exception` would imply a way more restrictive behavior than what we would desire.

An alias is not necessary as far as I can tell since you would only use this from IDE tasks i.e. you would only write it once. In those cases you do not want to use aliases. Aliases are only ok for one-time executions. Otherwise you come back to the code in X days and need to lookup what an alias stands for.